### PR TITLE
SUBMARINE-565. Documentation for Environment Management REST API

### PR DESCRIPTION
**Pull request by manirajv06**
### What is this PR for?
Documentation for Environment Management REST API's

### What type of PR is it?
Documentation

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-565

### How should this be tested?
* First time? Setup Travis CI as described on https://submarine.apache.org/contribution/contributions.html#continuous-integration
* Strongly recommended: add automated unit tests for any new or changed behavior
* Outline any manual steps to test the PR here.

### Questions:
* Does the licenses files need update? Yes/No
* Is there breaking changes for older versions? Yes/No
* Does this needs documentation? Yes/No

### CODE REVIEW

**Comment by pingsutw** [comment]
@manirajv06 The doc REST API is `/api/v1/environment`, but the current environment REST API is `/api/v1/environments`
Should we remove "s" in this line ?https://github.com/apache/submarine/blob/c9b9b32f8c320d1707acac342a20842d69644022/submarine-server/server-core/src/main/java/org/apache/submarine/server/rest/EnvironmentRestApi.java#L51

File: docs/userdocs/k8s/api/environment.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+> future releases.
+
+## Create Environment
+`POST /api/v1/environment`
+
+**Example Request**
+```sh
+curl -X POST -H "Content-Type: application/json" -d '
+{
+	"name": "my-submarine-env",
```
Comment (liuxunorg) [inline docs/userdocs/k8s/api/environment.md:32]: The indentation of json seems incorrect. Please use `2` or `4` spaces.

File: docs/userdocs/k8s/api/environment.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+	"kernelSpec" : {
+		"name" : "team_default_python_3.7",
+		"channels" : ["defaults"],
+		"dependencies" : 
+			["_ipyw_jlab_nb_ext_conf=0.1.0=py37_0", 
+  			"alabaster=0.7.12=py37_0",
+  			"anaconda=2020.02=py37_0",
+  			"anaconda-client=1.7.2=py37_0",
+  			"anaconda-navigator=1.9.12=py37_0"]
+	}
```
Comment (liuxunorg) [inline docs/userdocs/k8s/api/environment.md:43]: The indentation of json seems incorrect. Please use 2 or 4 spaces.
Other json files also have same problem.

**Comment by manirajv06** [comment]
Taken care

**Comment by pingsutw** [comment]
@manirajv06 I just fetch this pr, your email address is `mani@caastle.com`
As we discuss offline, you could change the email address in git config and force push again.
Hope these can help.
The PR looks good to me.
```
commit 22da1b3385ffca0bab5c5698edb00ba404244f56 (HEAD -> SUBMARINE-565)
Author: Manikandan R <mani@caastle.com>
Date:   Tue Jul 21 22:36:06 2020 +0530

    Addressed review comments

commit 46746743bcd0e82a4f9c0f3cab254284c3dbe89b
Author: Manikandan R <mani@caastle.com>
Date:   Thu Jul 16 10:59:35 2020 +0530

    Updated experiment doc with respect to environment management
```

**Pull request closed without merging**